packages/composites/ondo-calculated/src/lib/session.ts

```diff
@@ -0,0 +1,27 @@
+import { TZDate } from '@date-fns/tz'
+
+// Seconds relative to session boundary (-ve before, +ve after)
+export const calculateSecondsFromTransition = (
+  sessionBoundaries: string[],
```

Comment:
Is it acceptable that `sessionBoundaries` is an empty array? and if so that the expected output is `9007199254740991` rather than throwing an exception or sth? I can see that this is already in a test snapshot but just wanted to double check since I'm not familiar with the context.